rpc/jsonrpc/README.md

```diff
@@ -0,0 +1,66 @@
+# JsonRPC Example
+
+## Backend
+Dubbo3 provides Triple(Dubbo3), Dubbo2 protocols, which are native to the Dubbo framework.
```

Comment:
Dubbo3 provides Triple(Dubbo3) and Dubbo2 protocols, which are native to the Dubbo framework.
In addition, Dubbo3 also integrates a number of third-party protocols into Dubbo's programming and service governance architecture, including gRPC, Thrift, **JsonRPC**, Hessian2, and REST. The following describes the **JsonRPC** protocol example.

Reply:
I think [JSON](https://baike.baidu.com/item/JSON/2462549) RPC is right ,rather than JsonRPC

Reply:
yep, using JSON-RPC